.github/workflows/release.yaml

```
@@ -0,0 +1,64 @@
+name: Release
+
+on:
+  push:
+    #branches:
+    #  - master
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+
+    outputs:
+      version_tag: ${{ steps.tag-release.outputs.version_tag }}
+
+    steps:
+      - name: Protect parameters
+        uses: cultureamp/protect-event-parameters@v2
+        with:
+          allowlist: "originating_url,release_version,release_type"
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
```

Add this in as otherwise it runs in v12

```suggestion

      - uses: actions/setup-node@v3
        with:
          node-version: "18.x"

```

---

Doneski